drivers/sensor/icm42688/icm42688_common.c

```diff
@@ -69,6 +69,9 @@ int icm42688_configure(const struct device *dev, struct icm42688_cfg *cfg)
 	const struct icm42688_dev_cfg *dev_cfg = dev->config;
 	int res;
 
+	/* Disable interrupts, reconfigured at end */
+	res = icm42688_spi_single_write(&dev_cfg->spi, REG_INT_SOURCE0, 0);
```

Comment:
Adding new SPI transactions looks suspicious when the commit message says you're fixing register addresses and adding log messages. Can you update the commit message to explicitly call out that you're fixing data ready interrupts? You mention it in the PR summary but would be helpful to have that information in the commit history.